src/performance.py

```diff
@@ -26,9 +26,31 @@ def __init__(self, opus: Opus):
     def next_node(self):
         """Go to the next node."""
         current_node = self._nodes[self.history[-1]]
-        self.history.append(current_node.next)
-        self.emit("history-changed", self.history)
-
-        active_node = self._nodes[current_node.next]
-        for action_id in active_node.actions:
-            self.emit("run-action", action_id)
+        if isinstance(current_node.next, str):
+            print("Going to next node")
+            next_node = current_node.next
```

Comment:
Could we use `next_node_id` or similar here to be distinguish from an actual node object?

Reply:
Good idea, done